content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -683,6 +687,66 @@ More configuration options can be found on the [kong-plugin-ddtrace][3] plugin d
 [3]: https://github.com/DataDog/kong-plugin-ddtrace#configuration
 
 {{% /tab %}}
+
+{{% tab "Apache HTTP Server %}}
+
+## HTTPd with Datadog module
+Datadog provides a [module][1] for enhancing [Apache HTTP Server][2] and [IHS HTTP Server][3] capabilities with APM Tracing.
+
+### Compatibility
+Our module only support Apache HTTP Server `v2.4.x`.
+
+Since IHS HTTP Server is essentially a wrapper of the Appache HTTP Server, our module can also be used with IHS without any modifications.
+
+### Module installation
+<div class="alert alert-warning">
+  <strong>Note:</strong> Only Apache HTTP Server 2.4.x is supported.
+</div>
+
+The module is provided as a shared library for dynamical loading by HTTPd. Each supported platform
+and architecture has its own artifact hosted on [httpd-datadog's repository][1].
+
+Run the following script to download the latest version of the module:
+
+```bash
+curl -s https://api.github.com/repos/DataDog/httpd-datadog/releases/latest \
+| grep "mod_datadog-linux-x86_64.tar.gz" \
```

Comment:
If we are releasing only x86 (and so it seems https://github.com/DataDog/httpd-datadog/blob/main/.github/workflows/release.yml), then it is probably worth adding a warning above this snippet to make it explicit.
If we also release arm64, then we need to change this one-liner.

Reply:
addressed in 168c28c3efedb6b3a31c95f181ef04284b5adbd7